### ISSUE TITLE
Build a Feed-forward-neural-network for classification using Tensorflow

### ISSUE THREAD

**Issue by KrishnenduDakshi2002** [DSCKGEC/learn-machine-learn#6]
###**Requested Feature**

Build a Feed-forward-neural-network, compile and fit on data and in the output, f1-score, accuracy and confusion matrix must be printed, there is a function named metrics for printing the accuracy and confusion matrix. plot accuracy and loss curve for train data and validation data
Hyperparameter tuning or scalling can be done to improve the accuracy. As the dataset is imbalanced, prefer f1 score as metric while training.
Avoid under fitting and over fitting of  the model.

**Comment by aryabhatta0** [comment]
I would like to work on this issue. We can do it using keras from tensorflow, right?

**Comment by KrishnenduDakshi2002** [comment]
Yes we can do it.
 @aryabhatta0  issue is assigned.

**Comment by KrishnenduDakshi2002** [comment]
Any progress in this issue?

**Comment by aryabhatta0** [comment]
Have created a NN containing two hidden layers (10 nodes each) & got an accuracy of 92 over the test set. Considering ways to make it better. Should I create the PR now?

**Comment by KrishnenduDakshi2002** [comment]
> Have created a NN containing two hidden layers (10 nodes each) & got an accuracy of 92 over the test set. Considering ways to make it better. Should I create the PR now?

It is not a good accuracy compared to machine learning algorithms.
99% accuracy can easily be achieved on test set.
And don't make the model too complex( it won't need 10 nodes).
try scaling the data.
